lib/terjira/issue_cli.rb

```diff
@@ -109,6 +109,12 @@ def assign(*keys)
       show(issue.key_value)
     end
 
+    desc 'search [SUMMARY]', 'Search for an issue by summary'
+    def search(*keys)
+      search_term = client_class.search(summary: keys[0])
```

Comment:
I'm not sure that's a good idea. Most command-line utilities treat arguments separated with a space as 2 different ones. For example:

```bash
touch "file name" # creates file named 'file name'
touch file name # creates 2 files: 'file' and 'name'
```

have different effects and I'm not sure we would want to go against that standard.